data/ubuntu-distro-info.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
This cool new feature needs to be documented. 😉 

The documentation should explain:
- That we now support Ubuntu codenames.
- What's the correct format for the codenames: 
  - Do I need to use the full codename like `Focal Fossa` or can I use a part of the codename? Like just `focal` or `fossa`?

Reply:
The data source for tags/versions here is the docker one.
did a little research and basically the versioning is case sensitive (i.e. `FROM ubuntu:<codename>`) and requites only the first term of the codename phrase.